### ISSUE TITLE
Properly Handle a case when Text size is more than model handle

### ISSUE THREAD

**Issue by lalitpagaria** [obsei/obsei#164]
When this project was started initial intention was to handle only short text. But now we have added Google News and Crawlers, hence there is need to handle longer text as well.
As we know that most of BERT based model support 512 max tokens (with few exceptions like BigBird). Currently Analyzer ignore (#113) excessive text.

@akar5h Suggested nice ways to split based on tokenizer output instead of char size.

> So would it not be more optimal that instead of splitting texts, we split the tokenizer output ( tokenizer.batch_encode_plus output) like input_ids , attention mask into sequences of length 512 and feed these splits to the model ?

**Comment by lalitpagaria** [comment]
We can see if this can be incorporate here as suggested by @akar5h -
> https://colab.research.google.com/github/huggingface/notebooks/blob/master/examples/question_answering.ipynb#scrollTo=n9qywopnIrJH&uniqifier=1
> Scroll down to `Preprocessing the training data` , . Here we can see how huggingface tokenizer provides a way to split longer length documents as an inbuilt operation .
